packages/terra-site/src/examples/tabs/examples/IconOnlyTabs.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import IconBriefcase from 'terra-icon/lib/icon/IconBriefcase';
+import IconSearch from 'terra-icon/lib/icon/IconSearch';
+import IconBookmark from 'terra-icon/lib/icon/IconBookmark';
+import IconCalendar from 'terra-icon/lib/icon/IconCalendar';
+import Tabs from 'terra-tabs';
+import TabContent from './TabContentTemplate';
```

Comment:
nit: could we specify this import as `TabContentTemplate` so looking at the example people quickly know there was more setup than `<TabContent label="Search />`